src/enums/filingCodes.ts

```diff
@@ -1,8 +1,10 @@
+// FUTURE: add these to shared enum and import that instead
 export enum FilingCodes {
   ANNUAL_REPORT_BC = 'BCANN', // BCOMP - Annual Report
   ANNUAL_REPORT_OT = 'OTANN', // Others - Annual Report
   ADDRESS_CHANGE_BC = 'BCADD', // BCOMP - Change of Address
   ADDRESS_CHANGE_OT = 'OTADD', // Others - Change of Address
+  CONSENT_CONTINUATION_OUT = 'CONTO', // All entity types
```

Comment:
Should filing codes be a common component?  Are they used in the other repos too?

Reply:
Yes, filing codes are used in other UIs.